RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -812,10 +825,254 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
         return false;
     }
 
+    /**
+     * Parse the output of the SAM regripper plugin to get additional Account
+     * information
+     *
+     * @param regFilePath     the path to the registry file being parsed
+     * @param regAbstractFile the file to associate newly created artifacts with
+     *
+     * @return true if successful, false if parsing failed at some point
+     */
+    private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstractFile) {
+        File regfile = new File(regFilePath);
+        String parentModuleName = NbBundle.getMessage(this.getClass(), "ExtractRegistry.parentModuleName.noSpace");
+        SimpleDateFormat regRipperTimeFormat = new SimpleDateFormat("EEE MMM dd HH:mm:ss yyyy 'Z'");
+        regRipperTimeFormat.setTimeZone(getTimeZone("GMT"));
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(regfile))) {
+            SleuthkitCase tempDb = currentCase.getSleuthkitCase();
```

Comment:
Perhaps rename to skCase. That's the most common name we use for the Sleuthkit DB.